src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.js

```diff
@@ -277,6 +299,11 @@ export default class rd2ElevateCreditCardForm extends LightningElement {
         }
     }
 
+    paymentMethodEdited(newPaymentMethod) {
```

Comment:
could we change this function name to `isPaymentMethodChanged()`? We may want to come down on whether we want to use "edit" or "change" language in the codebase in regard to data changes to avoid any confusion. I think "change" is more often used for LWC UI/component state, vs "changed" is more used for field/data value changes.